boards/pinetime/Makefile

```diff
@@ -0,0 +1,4 @@
+MODULE = board
+DIRS = $(RIOTBOARD)/common/nrf52xxxdk
```

Comment:
Why is this pointed to the `nrf52xxxdk` and not `nrf52`? Is there some refactoring required in those directories?

Reply:
I was re-using the board.c. Anyhow, fixed.